### ISSUE TITLE
Thinking about an App/Project logo

### ISSUE THREAD

**Issue by RonjaKaehne** [diproche/webinterface#67]
We should collect ideas here for a own logo

**Comment by TimothyGillespie** [comment]
I don't think this should be a concern for us for now. The focuse should lie mainly on functionality. A logo can easily be inserted later. For now we can just use a simple big D as a place holder and change it up later.
